lib/gtfs/orm/stop_time.rb

```diff
@@ -9,8 +9,8 @@ class StopTime < Resource
       # ATTRIBUTES
 
       attribute :trip_id, String
-      attribute :arrival_time, Time
-      attribute :departure_time, Time
+      attribute :arrival_time, String
```

Comment:
GTFS providers we worked with included time strings with +24 hrs for arrivals on the following day (e.g. 35:15:00 for arrivals at 11:15 the following day) - these would fail to be parsed into a Time object.